presto-iceberg/src/main/java/com/facebook/presto/iceberg/CatalogType.java

```diff
@@ -20,6 +20,7 @@
 {
     HADOOP(HadoopCatalog.class.getName()),
     HIVE(HiveCatalog.class.getName()),
+    MEMORY(InMemoryCatalog.class.getName())
```

Comment:
Has the `InMemoryCatalog` merged into the Iceberg? If not, do you have a possible target ETA in your mind? In the future, I think we can directly import it from the Iceberg and avoid implementing the detailed catalog on the Presto side.

Reply:
just fyi, the `InMemoryCatalog` hasn't been merged yet (see https://github.com/apache/iceberg/pull/4054)